app/code/AKisilenko/ModuleLesson6/Console/Command/ProductQuantityUpdate.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace AKisilenko\ModuleLesson6\Console\Command;
+
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\CatalogInventory\Api\StockRegistryInterface;
+use Magento\CatalogInventory\Api\StockStateInterface;
+use Magento\Framework\App\State;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Magento\Framework\App\Area;
+
+class ProductQuantityUpdate extends Command
+{
+    /**
+     * @var State
+     */
+    private $state;
+
+    /**
+     * @var Magento\CatalogInventory\Api\StockStateInterface
+     */
+    protected $_stockStateInterface;
+    /**
+     * @var Magento\CatalogInventory\Api\StockRegistryInterface
+     */
+    protected $_stockRegistry;
+    /**
+     * CustomQtyUpdate constructor.
+     * @param ProductRepositoryInterface $productRepository
+     * @param StockStateInterface $stockStateInterface
+     * @param StockRegistryInterface $stockRegistry
+     * @param State $state
+     */
+    public function __construct(
+        ProductRepositoryInterface $productRepository,
+        StockStateInterface $stockStateInterface,
+        StockRegistryInterface $stockRegistry,
+        State $state
+    ) {
+        $this->productRepository = $productRepository;
+        $this->_stockStateInterface = $stockStateInterface;
+        $this->_stockRegistry = $stockRegistry;
+        $this->state = $state;
+        parent::__construct();
+    }
+
+    /**
+     * @param $productId
+     * @param $stockData
+     * @throws NoSuchEntityException
+     */
+    public function updateProductStock(
+        $productId,
+        $stockData
+    ) {
+        $product = $this->productRepository->getById($productId);
+        $stockItem = $this->_stockRegistry->getStockItem($product->getId());
+        $stockItem->setData('is_in_stock', 1);
+        $stockItem->setData('quantity', $stockData);
+        $stockItem->setData('manage_stock', 1);
+        $stockItem->setData('notify_quantity', 1);
```

Comment:
Incorrect field name 'notify_quantity'

Reply:
No, I mean 'use_config_notify_stock_qty'. Already fixed it